### PR TITLE
Ci/ruff

### PR DESCRIPTION
**Pull request by mfocko**
If I have to rebase this once again, I'll die rebasing…

### CODE REVIEW

**Review by nforro — APPROVED** [review]
Nice!

**Review by lbarcziova — APPROVED** [review]
nice!! :rocket:

**Comment by nforro** [comment]
> If I have to rebase this once again, I'll die rebasing…

Should I wait with merging https://github.com/packit/packit-service/pull/2588 then? :slightly_smiling_face:

File: ruff.toml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+select = [
+    "B",   # flake8-bugbear
+    "C4",  # flake8-comprehensions
+    "COM", # flake8-commas
+    "E",   # pycodestyle Error
+    "F",   # Pyflakes
+    "I",   # isort
+    "PERF",# Perflint
+    "PIE", # flake8-pie
+#    "PT",  # flake8-pytest-style, would be nice to enable
```
[the remark this answers is not in the document]
Comment (mfocko) [reply ruff.toml:10]: It’s much more painful than the rest, I could have a look later

**Pull request merged**